templates/web/hart/footer.html

```diff
@@ -7,18 +7,7 @@
     <div class="nav-wrapper clearfix">
         <div class="main-menu-wrapper">
             <div class="main-menu">
-            <ul class="clearfix">
-                <li class="home"><[% IF c.req.uri.path == '/' %]span[% ELSE %]a href="/"[% END %] class="report-a-problem-btn"
-                    >[% "Report" %]</[% c.req.uri.path == '/' ? 'span' : 'a' %]></li>[%
-                %]<li><[% IF c.req.uri.path == '/my' OR ( c.req.uri.path == '/auth' AND c.req.params.r == 'my' ) %]span[% ELSE %]a href="/my"[% END
```

Comment:
This tiny bit of functionality (also in Oxfordshire, others?) has been lost; I guess if it's /my or /auth (without any params) it could be a span rather than a link.